wled00/FX_fcn.cpp

```diff
@@ -131,6 +131,8 @@ void WS2812FX::service() {
 
   for(uint8_t i=0; i < MAX_NUM_SEGMENTS; i++)
   {
+    if (realtimeMode && useMainSegmentOnly && i == getMainSegmentId()) continue;
```

Comment:
This is ok, but we could also use freeze.
Freeze might be an easy way for keeping off non-main segments when live mode is entered from off as well!

Reply:
I have added freeze toggle to my UI (but is global).